sentry/values.yaml

```diff
@@ -8,12 +8,12 @@ user:
 images:
   sentry:
     repository: getsentry/sentry
-    tag: cc9f7d1
+    tag: 20.7.1
     pullPolicy: IfNotPresent
     # imagePullSecrets: []
   snuba:
     repository: getsentry/snuba
-    tag: 882be95ba0d462a29759a49b2e9aad0c1ce111a9
+    tag: e944bc414bfc97c9836febbfb2f310a3237f91cf
```

Comment:
`20.7.1` is also present. https://hub.docker.com/layers/getsentry/snuba/20.7.1/images/sha256-069d06ffa5ccb85324291e96de822e072663c55a26e5eb6f60164e3b8ae7d34e?context=explore

Reply:
I've tried this one and there seems to be an issue there.
The docker image is not accepting to run `api` through `./docker-entrypoint.sh api`

Reply:
I've seen those error messages as well.

Reply:
My errors (`AssertionError: All storage sets must be assigned to a cluster`) were also on that docker tag, btw